sessionctx/variable/session.go

```diff
@@ -956,6 +964,19 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		SetMaxDeltaSchemaCount(tidbOptInt64(val, DefTiDBMaxDeltaSchemaCount))
 	case TiDBUsePlanBaselines:
 		s.UsePlanBaselines = TiDBOptOn(val)
+	case TiDBIsolationReadEngines:
+		s.isolationReadEngines = make([]kv.StoreType, 0, 2)
+		if len(val) == 0 {
+			break
+		}
+		for _, engine := range strings.Split(val, ",") {
+			switch engine {
+			case "TiFlash":
```

Comment:
can we replace this magic string to `kv.TiFlash`?

Reply:
How can I make it? The `val` is always a string.